.aicoe-ci.yaml

```diff
@@ -1,2 +1,8 @@
 ---
 check: []
+build:
+  build-strategy: Source
+  base-image: registry.access.redhat.com/ubi9/nodejs-16:latest
```

Comment:
Have you tried `nodejs-16-minimal`?

Reply:
This indeed decreseases size of the image size without using minimal image is 1.01 GB and with minimal it is only 583 MB. But using minimal image leads to error shown on the picture below.
![image](https://user-images.githubusercontent.com/47832967/175934981-1c9b6770-7f11-4dfb-8613-2939981aa952.png)

tldr. From what I have gather it is an error made by ssh2 package when it is trying to optimize cipher ordering based on HW. More about it [here](https://github.com/backstage/backstage/issues/7523#issuecomment-945089084). It is used by dev dependency which is in the `yarn install --prodcution` removed so it should be safe to ignore